public/index.html

```diff
@@ -32,17 +39,11 @@
 		<!-- <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" /> -->
 		<title>Open Science Lab</title>
 	</head>
-	<!-- Google tag (gtag.js) -->
-	<script async src="https://www.googletagmanager.com/gtag/js?id=G-R9HWHJ5EVY"></script>
-	<script>
-	  window.dataLayer = window.dataLayer || [];
-	  function gtag(){dataLayer.push(arguments);}
-	  gtag('js', new Date());
-	
-	  gtag('config', 'G-R9HWHJ5EVY');
-	</script>
 	<body>
-		<noscript>You need to enable JavaScript to run this app.</noscript>
```

Comment:
Why did you remove this line?

Reply:
I thought it might not be very useful, but since it is kind of a rule, I put it back.